content/plugins/kenepa-resource-lock.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Resource Lock
+slug: resource-lock
+author_slug: kenepa
+categories: [panel-builder, panel-authorization]
+description: Filament Resource Lock is a Filament plugin that adds resource locking functionality to your site. When a user begins editing a resource, Filament Resource Lock automatically locks the resource to prevent other users from editing it at the same time. The resource will be automatically unlocked after a set period of time, or when the user saves or discards their changes.
+discord_url: https://discord.com/channels/883083792112300104/1094546390761341048
+docs_url: https://raw.githubusercontent.com/kenepa/resource-lock/2.x/README.md
+github_repository: kenepa/resource-lock
+has_dark_theme: true
+has_translations: true
+image: kenepa-resource-lock.jpg
+versions: [2, 3]
+publish_date: 2023-08-09
```

Comment:
Please fix this so that it is the date of the first release of your plugin, rather than the date you submitted it here.